service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RemoveOrderControllerAboutToSubmitTest.java

```diff
@@ -258,6 +263,50 @@ void shouldRemoveSDONoticeOfProceedingsAndSetStateToGatekeepingWhenRemovingASeal
         assertNull(responseData.getNoticeOfProceedingsBundle());
     }
 
+    @Test
+    void shouldRemoveDraftCaseManagementOrderAndRemoveHearingAssociation() {
+        UUID removedOrderId = UUID.randomUUID();
+        UUID additionalOrderId = UUID.randomUUID();
+        UUID hearingOrderBundleId = UUID.randomUUID();
+
+        Element<HearingOrder> caseManagementOrder1 = element(removedOrderId, HearingOrder.builder()
+            .status(DRAFT)
+            .type(HearingOrderType.DRAFT_CMO)
+            .build());
+
+        List<Element<HearingOrder>> caseManagementOrders = newArrayList(
+            caseManagementOrder1,
```

Comment:
could we rename this to orderToBeRemoved to make it a little clearer which one is being removed

Reply:
done